bin/ci-checks-js.sh

```diff
@@ -41,12 +46,17 @@ if [ "$CHECK_CORRECTNESS" = true ] ; then
   npm run lint || pFail
 fi
 
+# Notice we forward the tests output to a file.
+# That's because they generates 70k+ lines and we don't want to pollute the logs like that.
 if [ "$CHECK_TESTS" = true ] ; then
-  # we'll run the tests twich (once for each platform) if the platform env var is not set
-  if [[ -z "${TEST_RN_PLATFORM}" ]] ; then
-    TEST_RN_PLATFORM=android npm run test --maxWorkers=4 || pFail
-    TEST_RN_PLATFORM=ios npm run test --maxWorkers=4 || pFail
+  # we'll run the tests twice (once for each platform) if the platform env var is not set
+  if [[ -z "${TEST_RN_PLATFORM:-}" ]] ; then
+    echo "--- :microscope: :android: Unit tests"
+    TEST_RN_PLATFORM=android npm run test --maxWorkers=4 > android-tests-out.log || pFail
+    echo "--- :microscope: :ios: Unit tests"
+    TEST_RN_PLATFORM=ios npm run test --maxWorkers=4 > ios-tests-out.log || pFail
   else
-    npm run test --maxWorkers=4 || pFail
+    echo "--- :microscope: :$TEST_RN_PLATFORM: Unit tests"
+    npm run test --maxWorkers=4 > "$TEST_RN_PLATFORM-tests-out.log" || pFail
   fi
 fi
```

Comment:
I considered submitting these edits as a dedicated PR earlier, but given the intention of dismantling the script, I thought I'd just keep them here.